video/out/opengl/egl_helpers.c

```diff
@@ -111,13 +111,8 @@ static bool create_context(struct ra_ctx *ctx, EGLDisplay display,
         break;
     case 2:
         api = EGL_OPENGL_ES_API;
-        rend = EGL_OPENGL_ES2_BIT;
-        name = "GLES 2.x";
-        break;
-    case 3:
-        api = EGL_OPENGL_ES_API;
-        rend = EGL_OPENGL_ES3_BIT;
-        name = "GLES 3.x";
+        rend = EGL_OPENGL_ES2_BIT | EGL_OPENGL_ES3_BIT;
+        name = "GLES 2.x +";
```

Comment:
The latter `video: opengl: rework and remove ra_gl_ctx_test_version() ` seems to undo a lot of this. I agree that removing the explicit GLES 3 request is worth a separate commit though. I think it would probably be better to just go ahead and undo the switch case here and put some of the changes you did to the GLES logic in `video: opengl: rework and remove ra_gl_ctx_test_version()` here.

Reply:
Sure, can do. Are you thinking only about the `switch` -> `if`, or you want me to also move the `int es_version` -> `bool es` change as well?

Reply:
Yeah the int -> bool change should go in here as well.

Reply:
Thanks fixed.